src/EgoErpArchiver/ViewModel/TexturesWorkspaceViewModel.cs

```diff
@@ -65,13 +66,17 @@ public TexturesWorkspaceViewModel(MainViewModel mainView)
         public override void LoadData(object data)
         {
             ClearData();
+            List<ErpTextureViewModel> searchResults = new();
             foreach (var resView in ((ResourcesWorkspaceViewModel)data).Resources)
             {
                 if (resView.Resource.ResourceType == "GfxSRVResource")
                 {
-                    Textures.Add(new ErpTextureViewModel(resView));
+                    searchResults.Add(new ErpTextureViewModel(resView));
                 }
             }
+            searchResults.Sort((x, y) => string.Compare(x.DisplayName, y.DisplayName) );
+            foreach (ErpTextureViewModel rv in searchResults)
+                textures.Add(rv);
```

Comment:
You can use the `textureViewSource` like I did in the pssg editor to perform the sorting:
https://github.com/EgoEngineModding/Ego-Engine-Modding/blob/00ef7583dce9f7c859bc5b47b29377b8f829d0cb/src/EgoPssgEditor/ViewModel/TexturesWorkspaceViewModel.cs#L56